packages/fast-web-utilities/src/dom.ts

```diff
@@ -17,10 +17,10 @@ export function canUseFocusVisible(): boolean {
     }
 
     if (!canUseDOM()) {
```

Comment:
I think you still want to set `_canUseFocusVisible` to false so you avoid redundant `canUseDOM` calls

Reply:
@nicholasrice updated